### PR TITLE
add def_delegate for the deny command in the policy mixin to aws_s3_b…

### PR DESCRIPTION
**Pull request by ndavis-r7**
…ucket_policy.rb

### CODE REVIEW

File: lib/convection/model/template/resource/aws_s3_bucket_policy.rb [hunk 1]
```diff
@@ -17,6 +17,7 @@ class S3BucketPolicy < Resource
 
           def_delegators :@document, :allow, :id, :version, :statement
           def_delegator :@document, :name, :policy_name
+          def_delegators :@document, :deny, :id, :version, :statement
```
Comment (jmanero-r7) [inline lib/convection/model/template/resource/aws_s3_bucket_policy.rb]: You should only need to add to the list in line 18. e.g. ` def_delegators :@document, :allow, :deny, :id, :version, :statement`

**Comment by jmanero-r7** [comment]
This is also based off of the wrong branch

**Comment by ndavis-r7** [comment]
@jmanero-r7 what branch should it based off of?

File: lib/convection/model/template/resource/aws_s3_bucket_policy.rb (hunk 1 above)
Comment (ndavis-r7) [inline lib/convection/model/template/resource/aws_s3_bucket_policy.rb]: Didn't realize that, thanks

**Comment by jmanero-r7** [comment]
@ndavis-r7 Run `git rebase v0.2` in your working directory with `add_deny_to_s3_bucket_policy` checked out, then run `git push -f origin add_deny_to_s3_bucket_policy` and open a new PR

**Pull request closed without merging**